source/index.html

```diff
@@ -3,16 +3,15 @@
   <head>
     <title>My Awesome Style Editor</title>
     <link rel="stylesheet" type="text/css" href="main.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="styleEditor.js"></script>
+
   </head>
   <body>
     <div class="container">
       <form id="style_editor" name="style_editor">
-        <input name="selector" placeholder="css selector" />
-        <input name="property" placeholder="property" />
-        <input name="value" placeholder="value" />
-        <input type="submit" value="Style it!" />
+        <input name="selector" id="selector" placeholder="css selector" />
```

Comment:
Would have loved to see you do this without editting the HTML. An example selector that would have grabbed this element previously is `$('[name=selector]')`